scripts/generate_test_archive.js

```diff
@@ -17,6 +17,7 @@
         mainEntry = mainGroup.createEntry("test-entry-main");
 
     mainEntry.setProperty("username", "user123한@test.рф");
+    mainEntry.setAttribute("passExpiry", "30");
```

Comment:
The attribute name should be defined internally so that symbol can be used everywhere. Entry attributes are [defined here](https://github.com/buttercup/buttercup-core/blob/v0.43.0/source/system/Entry.js#L344).